client/src/components/Settings/actions.js

```diff
@@ -28,5 +28,31 @@ export const getSettings = () => {
                     error: `The user's current settings could not be obtained at this time.`
                 });
             });
-    }
-}
\ No newline at end of file
+    };
+};
+
+// Action creator to update current email
+
+export const updateEmail = (id, updatedEmail) => {
+    return dispatch => {
+        // Dispatch to reducer that current settings are being obtained from API
+        dispatch({type: FETCHING});
+        axios
+            // Update email on file in database
+            .put(`http://localhost:5000/api/users/${id}`, updatedEmail)
```

Comment:
Might change this to a `const` like `const BASE_URL = http://localhost:5000/api` so that it can be changed more easily later.

Reply:
Agree with Collen here